docs/guides/sparque-ai.md

```diff
@@ -33,6 +36,15 @@ In any case, the names of the configuration parameters must correspond exactly t
 Example for the specification of the SPARQUE configuration via `environment` file:
 
 ```ts
+// in case to use only selected features
+features: [
```

Comment:
I would list the features below the sparque configuration text block as it is in all other configuration examples below, first you configure sparque afterwards you define which feature you want to use